Ranorex_Browserstack/obj/x86/Debug/Mercy_Browserstack.csproj.FileListAbsolute.txt

```diff
@@ -0,0 +1,12 @@
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\bin\Debug\Ranorex_Browserstack.rxtmg
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\bin\Debug\Ranorex_Browserstack.rxtst
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\bin\Debug\Ranorex_Browserstack.exe.config
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\bin\Debug\Ranorex_Browserstack.xml
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\bin\Debug\Ranorex_Browserstack.exe
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\bin\Debug\Ranorex_Browserstack.pdb
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\bin\Debug\Ranorex.Core.Resolver.dll
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\obj\x86\Debug\Ranorex_Browserstack.csprojAssemblyReference.cache
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\obj\x86\Debug\Ranorex_Browserstack.csproj.CoreCompileInputs.cache
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\obj\x86\Debug\Ranorex_Browserstack.csproj.CopyComplete
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\obj\x86\Debug\Ranorex_Browserstack.exe
+C:\Users\siddhartha.p\Documents\Ranorex\RanorexStudio Projects\Ranorex_Browserstack\Ranorex_Browserstack\obj\x86\Debug\Ranorex_Browserstack.pdb

```

Comment:
Can we exclude this file? I think this would get regenerated when the user reopens the project

Reply:
Tested and removed the said file and all other unimportant files that were generated post build.
(Ranorex_Browserstack/obj/x86/Debug & Ranorex_Browserstack/bin/Debug)